GeoLib/AABB.h

```diff
@@ -108,9 +138,9 @@ class AABB
 	template <typename T>
 	bool containsPoint(T const & pnt) const
 	{
-		if (pnt[0] < _min_pnt[0] || _max_pnt[0] < pnt[0]) return false;
-		if (pnt[1] < _min_pnt[1] || _max_pnt[1] < pnt[1]) return false;
-		if (pnt[2] < _min_pnt[2] || _max_pnt[2] < pnt[2]) return false;
+		if (pnt[0] < _min_pnt[0] || _max_pnt[0] <= pnt[0]) return false;
+		if (pnt[1] < _min_pnt[1] || _max_pnt[1] <= pnt[1]) return false;
+		if (pnt[2] < _min_pnt[2] || _max_pnt[2] <= pnt[2]) return false;
```

Comment:
Why does ```<=``` return false? I don't think that is practical.  
Example: I want to check if a Point x is within the bounding box of a mesh. I get AABB and if x is located on the bottom of the mesh, the method returns true. If x is located on the top of the mesh, the method returns false.  
I would definitely prefer the original version.